tests/installation/disable_grub_timeout.pm

```diff
@@ -65,6 +65,7 @@ sub run {
     my $timeout = "-1";
     # SLE-12 GA only accepts positive integers in range [0,300]
     $timeout = "60" if is_sle('<12-SP1');
+    $timeout = "90" if (get_var("REGRESSION", '') =~ /xen|kvm|qemu/);
```

Comment:
Miss the original condition "is_sle('<12-SP1');"?

Reply:
No, the original `$timeout = "60" if is_sle('<12-SP1');` is still present.

Reply:
Yes, you are right. Sorry for missing that.